src/common/options.cc

```diff
@@ -1481,87 +1514,92 @@ std::vector<Option> get_global_options() {
 
     Option("mon_allow_pool_delete", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
     .set_default(false)
-    .set_description(""),
+    .set_description("allow pool deletions"),
 
     Option("mon_fake_pool_delete", Option::TYPE_BOOL, Option::LEVEL_ADVANCED)
     .set_default(false)
-    .set_description(""),
+    .set_description("fake pool deletions by renaming the rados pool"),
 
     Option("mon_globalid_prealloc", Option::TYPE_UINT, Option::LEVEL_ADVANCED)
     .set_default(10000)
-    .set_description(""),
+    .set_description("number of globalid values to preallocate")
+    .set_long_descriptions("This setting caps how many new clients can authenticate with the cluster before the monitors have to perform a write to preallocate more.  Large values burn through the 64-bit ID space more quickly."),
 
     Option("mon_osd_report_timeout", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(900)
-    .set_description(""),
+    .set_description("time before OSDs who do not report to the mons are marked down (seconds)"),
```

Comment:
Just wondering: since we're writing "OSDs" in all caps, shouldn't we do the same for "MONs" consistently?